src/chainparams.cpp

```diff
@@ -317,6 +321,8 @@ class CRegTestParams : public CChainParams {
         pchMessageStart[3] = 0xda;
         nDefaultPort = 18444;
         nPruneAfterHeight = 1000;
+        m_assumed_blockchain_size = 200;
```

Comment:
This sounds a bit excessive for regtest.

Reply:
do you have any suggestion for this value? thanks